dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt

```diff
@@ -153,11 +155,11 @@ Melee Ultimates deal maximum damage and or elevate other abilities. They typical
         - *Example: <:cane:535532878969438210> normally strikes in a 3x3 aoe; strikes in a 5x5 with*<:dragonriderlance:643161541931171872>
 .
 ⬥ **__Ability Damage__**:
-    • This is damage dealt with **abilities.**
-    • It is calculated with Auto attack damage value(s), weapon attack speed, strength level and other boosts.
-        - <:darkiceshard:915232154160594974> <:darkicesliver:915232153787334677> have Tier 88 ability damage: **~1637.65. **
-        - <:dragonriderlance:643161541931171872> has Tier 80 ability damage: **~1523.25**
-        - *With 99 strength, no boosts*
+    Base damage dealt with **abilities.**
+    It is calculated with weapon damage, weapon attack speed, strength level and other boosts.
+    • <:darkiceshard:915232154160594974> <:darkicesliver:915232153787334677> have Tier 88 ability damage: **~1637.65. **
+    • <:dragonriderlance:643161541931171872> has Tier 80 ability damage: **~1523.25**
+    • *With 99 strength, no boosts*
 
 **__Armour & Other Gear__**
 ⬥ T70 Bandos Power Armour <:bandoschest:643151021748518917><:Bandostassets:556592056953208852><:Bandosgloves:556588709890228252>
```

Comment:
I feel like CoE is a bit weird to use here considering the gear but I feel like that's slightly nitpicky